### DISCUSSION TITLE
Multiple buttons on Blitzwolf BW-SHP7

### DISCUSSION THREAD

**Discussion by cRaZy92** [arendst/Tasmota#16342] [Q&A] [answered]
Hello, I recently flash tasmota onto Blitzwolf BW-SHP7, everything works but there are 2 hardware buttons, 1 for each plug.

However, in device's tasmota page, there is only one ON/OFF state, 1 button and only one set of electricity stats (also HA reports only one, which makes sense).

I would need help with adding the second plug as input and stats for it. This is my first time every asking somewhere so I am not sure what should I provide, if there is anything that will help, just ask for it and I will gladly provide.

Thank you!

**Comment by barbudor** [comment, marked as answer]
You either didn't enter the [template](https://templates.blakadder.com/blitzwolf_SHP7.html) or forget to activate it
In any case, the power measurment is for both channel, this is an hardware limitation of the product, not a limitation of Tasmota

![image](https://user-images.githubusercontent.com/2996042/186774437-590fb28c-7fed-4b68-96aa-439d2e7f923c.png)

**Reply by cRaZy92** [reply]
Yes you are right, I didnt even noticed it. Thank you!
